sources/packages/backend/apps/api/src/route-controllers/announcements/_tests_/e2e/announcements.controller.getAnnouncements.e2e-spec.ts

```diff
@@ -0,0 +1,49 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { Repository } from "typeorm";
+import { Announcements } from "@sims/sims-db";
+import { createTestingAppModule } from "../../../../testHelpers";
+
+describe("AnnouncementsController(e2e)-getAnnouncements", () => {
+  let app: INestApplication;
+  let announcementsRepo: Repository<Announcements>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    announcementsRepo = dataSource.getRepository(Announcements);
+  });
+
+  it("Should return a saved announcement.", async () => {
+    // Arrange
+    const now = new Date();
+    const before = new Date(now.setFullYear(now.getFullYear() - 1));
+    const future = new Date(now.setFullYear(now.getFullYear() + 2));
```

Comment:
We can use the addDays() utility which we already have in place to add or remove days. 
![image](https://github.com/user-attachments/assets/c94eccd5-8b48-416c-bd92-cc708b7a795f)

Reply:
updated